app/models/event.rb

```diff
@@ -19,15 +18,25 @@ def status
     end
   end
 
+  def time_left
+    time_ago_in_words(self.start_time)
+  end
+
+  def ended_since
+    time_ago_in_words(self.end_time)
+  end
+
```

Comment:
@chelsejw can you confirm this file looks correct before we merge in? I'm fairly sure that `time_left` and `ended_since` were intended to be public methods so I shifted the lines around a bit when I was resolving the conflict.

Reply:
yes you're right shoould be public, thanks!